src/ui/tooltip.gd

```diff
@@ -32,11 +32,18 @@ func initialize(unit: Unit) -> void:
 	call_deferred("set_pos", unit)
 
 
+func add_extra_stats_text(unit: Unit, stat_name: String) -> String:
+	if(unit.extra_stats.get(stat_name, 0)):
+		return "(%+d)" % int(unit.extra_stats.get(stat_name))
+	return ""
+
+
 func set_pos(unit: Unit) -> void:
 	var position_offset := Vector2(-100, -100)
 	if unit.grid_position.y < 3:
 		position_offset.y += vert_offset
 	else:
 		position_offset.y -= vert_offset - 5
 	position = unit.position + position_offset
-	position.x = clampf(position.x, -30, 670)
+	position.x = clampf(position.x, -100 + size.x / 2, 740 - size.x / 2)
```

Comment:
I really wish there was a better way for this, not worth the effort right now though.